rbe.bazelrc

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
Please delete this line. Run `hack/verify_boilerplate.py` to ensure this is all correct